### PR TITLE
Update docs and example with record file

### PR DESCRIPTION
**Pull request by codemonkey9000**
* Added record data to file section in the Simulator Reference
* Added log and record options help text to Simulator Reference
* Updated find-the-center example with new code for record to Examples page

@oleiman can you let me know if this is sufficient or if we could add any more text to the Record Data to File section about why you would want to use this feature?

PR Review Checklist:
- [x] Test search functionality on desktop
- [x] Test search functionality on mobile
- [x] Test filter functionality
- [x] Test mobile header navigation
- [x] Test desktop header navigation
- [x] Test desktop homepage navigation
- [x] Run `npm run check-links` with 0 broken links
- [ ] Run `npm run write-good <files being edited>` to "lint" the docs

### CODE REVIEW

File: source/includes/_simulator-reference.html.md [hunk 1]
```diff
@@ -54,16 +54,16 @@ Note that config in __main__ is the brain configuration and remains the same thr
...
+                 [--predict [PREDICT]] [--verbose] [--performance]
+                 [--log LOG [LOG ...]] [--record RECORD]
 
 optional arguments:
   -h, --help            show this help message and exit
   --accesskey ACCESSKEY
                         The access key to use when connecting to the BRAIN
                         server. If specified, it will be used instead of any
-                        access key information stored in a bonsai config file.
+                        access keyinformation stored in a bonsai config file.
```
Comment (oleiman) [inline source/includes/_simulator-reference.html.md]: lost some whitespace

File: source/includes/_simulator-reference.html.md [hunk 2]
```diff
@@ -147,6 +156,39 @@ Python <simulator_file>.py --predict=1
...
+    
+    config->set_record_enabled(true);
+    config->set_record_file("find_the_center.json");
+
+    brain = bonsai_ai.Brain(config)
+    sim = BasicSimulator(brain, "find_the_center_sim")
+    sim.enable_keys(["delta_t", "goal_count"], "ftc")
+
+    print('starting...')
+    last = clock() * 10000000
```
Comment (oleiman) [inline source/includes/_simulator-reference.html.md]: should be `1000000` (probably a typo carried over during copy pasta)

File: source/includes/examples/_find-the-center.html.md [hunk 3]
```diff
@@ -137,16 +139,36 @@ class BasicSimulator(bonsai_ai.Simulator):
...
+    # recording to CSV.
+    # config->set_record_enabled(true);
+    # config->set_record_file("find_the_center.json");
+
     brain = bonsai_ai.Brain(config)
     sim = BasicSimulator(brain, "find_the_center_sim")
+    sim.enable_keys(["delta_t", "goal_count"], "ftc")
 
     print('starting...')
+    last = clock() * 10000000
```
Comment (oleiman) [inline source/includes/examples/_find-the-center.html.md]: there it is again (should be 10^6)

File: source/includes/_simulator-reference.html.md (hunk 1 above)
Comment (codemonkey9000) [reply source/includes/_simulator-reference.html.md]: Yeah this unfortunately is in the CLI but I have been too lazy to put in a Diff so I was just changing it in the docs. :P

File: source/includes/_simulator-reference.html.md (hunk 2 above)
Comment (codemonkey9000) [reply source/includes/_simulator-reference.html.md]: I'll open a diff to fix this in the codebase.

**Comment by codemonkey9000** [comment]
Okay @oleiman how does this look now?

**Pull request merged**